interfaces/builtin/timeserver_control.go

```diff
@@ -79,6 +89,9 @@ dbus (receive)
 # timedatectl's set-ntp command.
 /usr/bin/timedatectl{,.real} ixr,
 
+# timedatectl needs to bind the client side of the socket
+unix (bind) type=stream addr="@*/bus/timedatectl.rea/system",
```

Comment:
fyi, this is from `bus_socket_connect()`, but is also generated at runtime and uses the process' command, see https://github.com/systemd/systemd/blob/b01f00e9c5409771fb5eacc018d2b49bc002f67d/src/libsystemd/sd-bus/bus-socket.c#L986-L998
since we allow calling /usr/bin/timedatectl{,.real}, the pattern should likely be
```suggestion
unix (bind) type=stream addr="@*/bus/timedatectl*/system",
```

Reply:
or not, timedatectl is script redirecting to timedatectl.real if needed:
```
#!/bin/sh

# wrapper to make sure timedatectl works with the
# writable-paths setup on ubuntu core (making the symlink
# absolute instead of using a relative path)
# LP: #1650688 for details
...
TIMEDATECTL=/usr/bin/timedatectl.real
...
    *)
        $TIMEDATECTL "$@"
        ;;
esac
```

Reply:
Thanks, modified as suggested to make it future proof.